### ISSUE TITLE
Translation and Documentation Missing

### ISSUE THREAD

**Issue by perlrocks** [SYSTRAN/resources-api-python-client#1]
I'm not sure how to use this since I couldn't find any documentation, but looking at your test case, I tried the following:

```
source = "en"
target = "fr"
inputs = ["hello"]
result = dictionary_api.resources_dictionary_lookup_get(source=source, target=target, input=inputs)
print result.__repr__()
```

The translation was missing from the output:

`<systran_resources_api.models.lookup_response outputs=[<systran_resources_api.models.lookup_results output=<systran_resources_api.models.lookup_output_object auto_complete=None matches=[<systran_resources_api.models.lookup_match_object target=None source=<systran_resources_api.models.lookup_source_object info='' term='hello' inflection='' pos='intj' lemma='hello' phonetic=''> auto_complete=False other_expressions=None model_name='mono-enfr.mod'>, <systran_resources_api.models.lookup_match_object target=None source=<systran_resources_api.models.lookup_source_object info='' term='hello' inflection='(pl:hello)' pos='noun' lemma='hello' phonetic=u'[h\u025b\u02c8lo\u028a\u032f]'> auto_complete=False other_expressions=None model_name='mono-enfr.mod'>] s_dict_seach=None>>] warning=None error=None>`

**Comment by pbrunelle-systran** [comment]
Have you the same result with others word.  Try "plane".

**Comment by jlorieux-systran** [comment]
Hi,

Using the following code, we reproduce your issue
```python
#!/usr/bin/env python
# coding: utf-8

import os

import systran_resources_api
import systran_resources_api.configuration

api_key_file = os.path.join(os.path.dirname(__file__), "../", "api_key.txt")
systran_resources_api.configuration.load_api_key(api_key_file)
api_client = systran_resources_api.ApiClient()
dictionary_api = systran_resources_api.DictionaryApi(api_client)

source = "en"
target = "fr"
inputs = ["hello"]
result = dictionary_api.resources_dictionary_lookup_get(source=source, target=target, input=inputs)

print result.__repr__()
```

Output with "hello" as input:
```python
<systran_resources_api.models.lookup_output_object 
  auto_complete=None 
  matches=[
    <systran_resources_api.models.lookup_match_object 
      target=None 
      source=<systran_resources_api.models.lookup_source_object
        info=''
        term='hello'
        inflection=''
        pos='intj'
        lemma='hello'
        phonetic=''
      > 
      auto_complete=False
      other_expressions=None
      model_name='mono-enfr.mod'
    >, 
    <systran_resources_api.models.lookup_match_object 
      target=None 
      source=<systran_resources_api.models.lookup_source_object
        info=''
        term='hello'
        inflection='(pl:hello)'
        pos='noun'
        lemma='hello'
        phonetic=u'[h\u025b\u02c8lo\u028a\u032f]'
      >
      auto_complete=False
      other_expressions=None
      model_name='mono-enfr.mod'
    >
  ]
  s_dict_seach=None
>
```


Output with "plane" as input:
```python
<systran_resources_api.models.lookup_output_object
  auto_complete=None
  matches=[
    <systran_resources_api.models.lookup_match_object
      target=None
      source=<systran_resources_api.models.lookup_source_object
        info=''
        term='plane'
        inflection=''
        pos='adj'
        lemma='plane'
        phonetic=u'[ple\u026an]'
      >
      auto_complete=False
      other_expressions=None
      model_name='mono-enfr.mod'
    >,
    <systran_resources_api.models.lookup_match_object
      target=None
      source=<systran_resources_api.models.lookup_source_object
        info=''
        term='plane'
        inflection='(pl:planes)'
        pos='noun'
        lemma='plane'
        phonetic=u'[ple\u026an]'
      >
      auto_complete=False
      other_expressions=[
        <systran_resources_api.models.lookup_other_expression
          source='horizontal plane'
          target='plan horizontal'
          context=''
        >,
        <systran_resources_api.models.lookup_other_expression
          source='vertical plane'
          target='plan vertical'
          context=''
        >,
        <systran_resources_api.models.lookup_other_expression
          source='viewing plane'
          target='plan visuel'
          context=''
        >,
        <systran_resources_api.models.lookup_other_expression
          source='plane of the image receptor'
          target=u"plan du r\xe9cepteur d'image"
          context=''
        >
      ]
      model_name='mono-enfr.mod'
    >,
    <systran_resources_api.models.lookup_match_object
      target=None
      source=<systran_resources_api.models.lookup_source_object
        info=''
        term='plane'
        inflection='(planes/planed)'
        pos='verb'
        lemma='plane'
        phonetic=u'[ple\u026an]'
      >
      auto_complete=False
      other_expressions=None
      model_name='mono-enfr.mod'
    >
  ]
  s_dict_seach=None
>
```

Translation is correctly set for "other expressions", but not for the word itself.
We will investigate and keep you informed.

**Comment by jlorieux-systran** [comment]
The missing translation is fixed in [v1.0.3](https://github.com/SYSTRAN/resources-api-python-client/releases/tag/v1.0.3)
